deploy/fat_container/scripts/renew-certificate.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+set -o allexport
+. /opt/appsmith/docker.env
+set +o allexport
+if [[ -n $CUSTOM_DOMAIN ]]; then
+    #then run script
+	data_path="/opt/appsmith/data/certificate"
+	domain="$CUSTOM_DOMAIN"
+	rsa_key_size=4096
+	
+	certbot certonly --webroot --webroot-path="$data_path/certbot" \
+		--register-unsafely-without-email \
+		--domains $domain \
+		--rsa-key-size $rsa_key_size \
+		--agree-tos \
+		--force-renewal
+	supervisorctl restart editor
+fi
```

Comment:
Let's add a little note when custom domain is not configured, since this script effectively won't do anything in that case. When we have more docs on this topic, we can even share link to the relevant page in the output here.

```suggestion
else
  echo 'Custom domain not configured. Cannot enable SSL without a custom domain.' >&2
fi

```